ta_services/ta_services.mk

```diff
@@ -0,0 +1,15 @@
+ta_dirs := keymaster
+
+ta_flags := CROSS_COMPILE="$(CROSS_COMPILE)" \
+	    TA_DEV_KIT_DIR="$(CURDIR)/$(out-dir)/export-$(sm)"
+
+.PHONY: ta_services
+ta_services: ta_dev_kit
+	${q}$(foreach dir,$(ta_dirs), \
+		$(MAKE) -C ta_services/$(dir) $(ta_flags) \
+		O=$(CURDIR)/$(out-dir)/ta_services/$(dir) &&) true
+
+# set propercleandirs
```

Comment:
Seems like cleaning is missing

Reply:
Not sure how to do it properly. As I understood, I should provide the list of files to cleandirs var, but the problem is that creation of these files are done by a separate independent Makefile (`ta_services/keymaster/Makefile`). As an option, I can introduce a new target isntead (`ta_services_clean`), that will do the same as ta_services, but call invoke make with clean target for each ta.

Reply:
Sounds good with a new target

Reply:
fixed